docs/get-started/python.md

```diff
@@ -0,0 +1,124 @@
+
+### Copy the code sample
+1. Copy and paste the following code sample into the existing `main.py` file of your project.
+```python
+#!/usr/bin/env python
+from constructs import Construct
+from cdk8s import App, Chart
+from imports import k8s
+
+class MyAwesomeApp(Chart):
+    def __init__(self, scope: Construct, id: str):
+        super().__init__(scope, id)
+
+        # Define a Kubernetes Service
+        k8s.KubeService(self, "my-service",
+            spec=k8s.ServiceSpec(
+                selector={"app": "my-app"},
+                ports=[k8s.ServicePort(port=80, target_port=k8s.IntOrString.from_number(8080))],
+                type="LoadBalancer"
+            )
+        )
+
+app = App()
+MyAwesomeApp(app, "python-service")
+
+app.synth()
+```
+
+A few things worth noting about this sample:
+
+* The `__init__` method of the `MyAwesomeApp` class creates a Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) by instantiating the `k8s.KubeService` class with the following parameters: `self` as the scope of the Service resource within the `MyAwesomeApp` class, a string identifier "my-service", and a `k8s.ServiceSpec` object that defines the Service specification with a selector dictionary having the key "app" and the value "my-app", a list of ports containing a single `k8s.ServicePort` object with a port set to 80 and a target_port set to 8080, and a `type` parameter with the value "LoadBalancer".
+* An instance of the `MyAwesomeApp` class is created with the identifier "python-service".
+
+## Generating Kubernetes manifests
+After you have defined the Kubernetes resources for your project, you are ready to generate the Kubernetes manifest that will define your Service resource. 
+
+### Run the synth command
+1. Open a terminal and navigate to your project directory.
+2. Run the [synth](https://cdk8s.io/docs/latest/cli/synth/) command. This command generates a Kubernetes manifest file in the `dist` folder of your project directory. The manifest file is based on the parameters specified in your application.
+```console
+cdk8s synth
+```
+
+### View the manifest
+1. Open the `dist/go-service.k8s.yaml` file. You should see a manifest similar to the following:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+ name: go-service-service-c8d5f888
+spec:
+ ports:
+   - port: 80
+     targetPort: 8080
+ selector:
+   app: my-app
+ type: LoadBalancer
+```
+
+## Wrap up
+- To continue your journey, develop a complete Kubernetes [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) and [Service](https://kubernetes.io/docs/concepts/services-networking/service/) using the [Hello, Kubernetes!](https://github.com/cdk8s-team/cdk8s/tree/master/examples/python/hello) sample application.
```

Comment:
The new [examples repo](https://github.com/cdk8s-team/cdk8s-examples) is now ready btw.

Reply:
Awesome, I'll update.